File: e-commerce.js
class RecommendationSystem {
    constructor(userData) {
        this.userData = userData; 
    }
    getRecommendations(userId) {
    
    }
}

const userData = {
    "user1": ["Audi", "Toyota", "Mitsubishi"],
    "user2": ["apples", "mangoes", "banana"],
};
const recommendationSystem = new RecommendationSystem(userData);
const userId = "user1";
const recommendations = recommendationSystem.getRecommendations(userId);
console.log(recommendations); 

File: e-commerce.py
user_preferences = {'banana'}

def add_user(user_id, preferred_categories):

    user_preferences[user_id] = preferred_categories


def get_recommendations(user_id):

    preferred_categories = user_preferences.get(user_id, [])

    if not preferred_categories:

        return []


    return ["Recommendation1", "Recommendation2", "Recommendation3"]
